File: ShipmentControlbackend/src/main/java/com/isdmoldova/shipmentcontrolbackend/dto/RouteDTO.java
package com.isdmoldova.shipmentcontrolbackend.dto;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class RouteDTO {

    private Long userId;
    private Long routeId;
    private List<TransportDTO> transportDTOList;
    private ItineraryDTO itineraryDTO;
    private List<AvailableDaysRentDTO> availableDaysRentList;
    private Double maximalLoadWeight;
    private Double maxLoadVolume;
    private Long estimatedAmountTimeShipment;
    private String routeDescription;
    private String providerCompany;

}

File: ShipmentControlfrontend/src/app/model/route.dto.ts
import {TransportDto} from "./transport.dto";
import {ItineraryDto} from "./itinerary.dto";
import {AvailableDaysRentDto} from "./availableDaysRentDto";


export interface RouteDto {
  routeId: number;
  userId: number;
  transportDTOList: TransportDto[];
  itineraryDTO: ItineraryDto;
  availableDaysRentList: AvailableDaysRentDto[];
  maximalLoadWeight: number;
  maxLoadVolume: number;
  estimatedAmountTimeShipment: number;
  routeDescription: string;
  providerCompany: string;
}

